<ui version="4.0" >
 <class>TeXDocument</class>
 <widget class="QMainWindow" name="TeXDocument" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>580</height>
   </rect>
  </property>
  <property name="cursor" >
   <cursorShape>IBeamCursor</cursorShape>
  </property>
  <property name="windowTitle" >
   <string>untitled.tex[*]</string>
  </property>
  <property name="autoFillBackground" >
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground" >
    <bool>false</bool>
   </property>
   <layout class="QVBoxLayout" >
    <property name="leftMargin" >
     <number>0</number>
    </property>
    <property name="topMargin" >
     <number>0</number>
    </property>
    <property name="rightMargin" >
     <number>0</number>
    </property>
    <property name="bottomMargin" >
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter" >
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTextEdit" name="textEdit" >
       <property name="frameShape" >
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Plain</enum>
       </property>
       <property name="tabStopWidth" >
        <number>36</number>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_console" >
       <property name="frameShape" >
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Plain</enum>
       </property>
       <property name="midLineWidth" >
        <number>0</number>
       </property>
       <property name="readOnly" >
        <bool>true</bool>
       </property>
       <property name="tabStopWidth" >
        <number>36</number>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>780</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actionNew" />
    <addaction name="actionOpen" />
    <addaction name="actionOpen_Recent" />
    <addaction name="separator" />
    <addaction name="actionClose" />
    <addaction name="actionSave" />
    <addaction name="actionSave_As" />
    <addaction name="actionRevert_to_Saved" />
   </widget>
   <widget class="QMenu" name="menuEdit" >
    <property name="title" >
     <string>Edit</string>
    </property>
    <addaction name="actionUndo" />
    <addaction name="actionRedo" />
    <addaction name="separator" />
    <addaction name="actionCut" />
    <addaction name="actionCopy" />
    <addaction name="actionPaste" />
    <addaction name="actionClear" />
    <addaction name="separator" />
    <addaction name="actionSelect_All" />
   </widget>
   <widget class="QMenu" name="menuRun" >
    <property name="title" >
     <string>Typeset</string>
    </property>
    <addaction name="actionTypeset" />
    <addaction name="separator" />
    <addaction name="actionTeX" />
    <addaction name="actionLaTeX" />
   </widget>
   <widget class="QMenu" name="menuWindow" >
    <property name="title" >
     <string>Window</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSearch" >
    <property name="title" >
     <string>Search</string>
    </property>
    <addaction name="actionFind" />
    <addaction name="actionFind_Again" />
    <addaction name="actionReplace" />
    <addaction name="separator" />
    <addaction name="actionCopy_to_Find" />
    <addaction name="actionCopy_to_Replace" />
    <addaction name="actionFind_Selection" />
    <addaction name="separator" />
    <addaction name="actionGo_to_Line" />
   </widget>
   <widget class="QMenu" name="menuFormat" >
    <property name="title" >
     <string>Format</string>
    </property>
    <addaction name="actionFont" />
    <addaction name="separator" />
    <addaction name="actionComment" />
    <addaction name="actionUncomment" />
    <addaction name="actionIndent" />
    <addaction name="actionUnindent" />
    <addaction name="separator" />
    <addaction name="actionWrap_Lines" />
   </widget>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>Help</string>
    </property>
    <addaction name="actionAbout_QTeX" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuEdit" />
   <addaction name="menuSearch" />
   <addaction name="menuFormat" />
   <addaction name="menuRun" />
   <addaction name="menuWindow" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QToolBar" name="toolBar_basic" >
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <property name="allowedAreas" >
    <set>Qt::LeftToolBarArea|Qt::NoToolBarArea|Qt::TopToolBarArea</set>
   </property>
   <property name="iconSize" >
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="toolButtonStyle" >
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionTypeset" />
   <addaction name="separator" />
   <addaction name="actionNew" />
   <addaction name="actionOpen" />
   <addaction name="actionSave" />
   <addaction name="separator" />
   <addaction name="actionUndo" />
   <addaction name="actionRedo" />
   <addaction name="separator" />
   <addaction name="actionCut" />
   <addaction name="actionCopy" />
   <addaction name="actionPaste" />
  </widget>
  <widget class="QStatusBar" name="myStatusBar" >
   <property name="font" >
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <action name="actionNew" >
   <property name="icon" >
    <iconset resource="resources.qrc" >:/images/images/filenew.png</iconset>
   </property>
   <property name="text" >
    <string>New</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen" >
   <property name="icon" >
    <iconset resource="resources.qrc" >:/images/images/fileopen.png</iconset>
   </property>
   <property name="text" >
    <string>Open...</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose" >
   <property name="text" >
    <string>Close</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionSave" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="resources.qrc" >:/images/images/filesave.png</iconset>
   </property>
   <property name="text" >
    <string>Save</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As" >
   <property name="text" >
    <string>Save As...</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionRevert_to_Saved" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Revert to Saved</string>
   </property>
  </action>
  <action name="actionUndo" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="resources.qrc" >:/images/images/editundo.png</iconset>
   </property>
   <property name="text" >
    <string>Undo</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="resources.qrc" >:/images/images/editredo.png</iconset>
   </property>
   <property name="text" >
    <string>Redo</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionCut" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="resources.qrc" >:/images/images/editcut.png</iconset>
   </property>
   <property name="text" >
    <string>Cut</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="resources.qrc" >:/images/images/editcopy.png</iconset>
   </property>
   <property name="text" >
    <string>Copy</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="resources.qrc" >:/images/images/editpaste.png</iconset>
   </property>
   <property name="text" >
    <string>Paste</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionClear" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Clear</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Backspace</string>
   </property>
  </action>
  <action name="actionTypeset" >
   <property name="icon" >
    <iconset resource="resources.qrc" >:/images/images/typeset.png</iconset>
   </property>
   <property name="text" >
    <string>Typeset</string>
   </property>
  </action>
  <action name="actionTeX" >
   <property name="text" >
    <string>TeX</string>
   </property>
  </action>
  <action name="actionLaTeX" >
   <property name="text" >
    <string>LaTeX</string>
   </property>
  </action>
  <action name="actionFind" >
   <property name="text" >
    <string>Find...</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionFind_Again" >
   <property name="text" >
    <string>Find Again</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionReplace" >
   <property name="text" >
    <string>Replace...</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionCopy_to_Find" >
   <property name="text" >
    <string>Copy to Find</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionCopy_to_Replace" >
   <property name="text" >
    <string>Copy to Replace</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Shift+E</string>
   </property>
  </action>
  <action name="actionGo_to_Line" >
   <property name="text" >
    <string>Go to Line...</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionFind_Selection" >
   <property name="text" >
    <string>Find Selection</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionSelect_All" >
   <property name="text" >
    <string>Select All</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionPdfTeX" >
   <property name="text" >
    <string>pdfTeX</string>
   </property>
  </action>
  <action name="actionXeTeX" >
   <property name="text" >
    <string>XeTeX</string>
   </property>
  </action>
  <action name="actionLuaTeX" >
   <property name="text" >
    <string>luaTeX</string>
   </property>
  </action>
  <action name="actionFont" >
   <property name="text" >
    <string>Font...</string>
   </property>
  </action>
  <action name="actionIndent" >
   <property name="text" >
    <string>Indent</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+]</string>
   </property>
  </action>
  <action name="actionUnindent" >
   <property name="text" >
    <string>Unindent</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+[</string>
   </property>
  </action>
  <action name="actionComment" >
   <property name="text" >
    <string>Comment</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Shift+]</string>
   </property>
  </action>
  <action name="actionUncomment" >
   <property name="text" >
    <string>Uncomment</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Shift+[</string>
   </property>
  </action>
  <action name="actionWrap_Lines" >
   <property name="text" >
    <string>Wrap Lines</string>
   </property>
  </action>
  <action name="actionNone" >
   <property name="text" >
    <string>None</string>
   </property>
  </action>
  <action name="actionAbout_QTeX" >
   <property name="text" >
    <string>About TeXWorks...</string>
   </property>
  </action>
  <action name="actionOpen_Recent" >
   <property name="text" >
    <string>Open Recent</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>actionUndo</sender>
   <signal>activated()</signal>
   <receiver>textEdit</receiver>
   <slot>undo()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>387</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRedo</sender>
   <signal>activated()</signal>
   <receiver>textEdit</receiver>
   <slot>redo()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>387</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCut</sender>
   <signal>activated()</signal>
   <receiver>textEdit</receiver>
   <slot>cut()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>387</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCopy</sender>
   <signal>activated()</signal>
   <receiver>textEdit</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>387</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPaste</sender>
   <signal>activated()</signal>
   <receiver>textEdit</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>387</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSelect_All</sender>
   <signal>activated()</signal>
   <receiver>textEdit</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>387</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>textEdit</sender>
   <signal>undoAvailable(bool)</signal>
   <receiver>actionUndo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>387</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>textEdit</sender>
   <signal>redoAvailable(bool)</signal>
   <receiver>actionRedo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>387</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>textEdit</sender>
   <signal>copyAvailable(bool)</signal>
   <receiver>actionCopy</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>387</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>textEdit</sender>
   <signal>copyAvailable(bool)</signal>
   <receiver>actionCut</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>387</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
